spec/features/user_login_logout_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+require "./spec/support/user_login.rb"
+
+RSpec.describe "User login and logout" do
+  let!(:user) { User.create(name: "Markus",
+                           email: "molsen13@gmail.com",
+                           password: "password") }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 7, 50 should be on its own line.